local/compose.go

```diff
@@ -1086,24 +1086,24 @@ func (s *composeService) ensureVolume(ctx context.Context, volume types.VolumeCo
 	// TODO could identify volume by label vs name
 	_, err := s.apiClient.VolumeInspect(ctx, volume.Name)
 	if err != nil {
-		if errdefs.IsNotFound(err) {
-			eventName := fmt.Sprintf("Volume %q", volume.Name)
-			w := progress.ContextWriter(ctx)
-			w.Event(progress.CreatingEvent(eventName))
-			// TODO we miss support for driver_opts and labels
-			_, err := s.apiClient.VolumeCreate(ctx, mobyvolume.VolumeCreateBody{
-				Labels:     volume.Labels,
-				Name:       volume.Name,
-				Driver:     volume.Driver,
-				DriverOpts: volume.DriverOpts,
-			})
-			if err != nil {
-				w.Event(progress.ErrorEvent(eventName))
-				return err
-			}
-			w.Event(progress.CreatedEvent(eventName))
+		if !errdefs.IsNotFound(err) {
```

Comment:
I'm not sure I understand what changed in the code. I only see that this condition changed but we are returning the same `err` no?